web-app/src/test/java/com/google/sps/data/QuadTreeTest.java

```diff
@@ -75,4 +77,55 @@ public void stopAtMaxDepth() throws IOException {
     }
   }
 
+  // Query Tests
+  QuadTree simpleTree;
+
+  @Before
```

Comment:
I was under the impression that you can only have one @Before per test class (so both setUp and simpleTree will be called before every function). Maybe we should split insert and query tests?